src/acl/agent.ts

```diff
@@ -418,7 +420,7 @@ function removeAgentFromDefaultRule(
   return [ruleWithoutAgent, ruleForOtherTargets];
 }
 
-function intialiseAclRule(access: unstable_Access): unstable_AclRule {
+export function initialiseAclRule(access: unstable_Access): unstable_AclRule {
```

Comment:
It we're re-using these for groups and classes, we should probably move these to `./acl.ts`, since they're not agent-specific.

Also, given that exporting them makes them available to client libraries, we should prefix their names with `internal_`.

Reply:
Ditto to above.